sut/src/docker-compose.retorch.yml

```diff
@@ -173,15 +173,15 @@ services:
     volumes:
       - ./ApiGateways/Envoy/config/mobileshopping:/etc/envoy
     ports:
-    - "5200:80"
-    - "15200:8001"
+      - "5200:80"
```

Comment:
General: avoid mixing formatting changes and functional changes, specially in large PRs like this. See https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md

Reply:
I apologize; I edited this file to enable networking between the System Under Test (SUT), the slave, and Selenoid. Unfortunately, I had to adjust the indentation in the ports.